.travis.yml

```diff
@@ -1,23 +1,25 @@
 language: python
-python: 2.7
-sudo: false
+sudo: true
 cache:
+  pip: true
   directories:
-  - buildout-cache
+    - eggs
+    - downloads
+python:
+  - 2.7.13
 env:
-  - PLONE_VERSION=4.2
```

Comment:
if you're not removing support for Plone 4.2 why did removed this line?

it's totally valid to remove support for Plone 4.3 but you need to add that to the changelog.